modules/ROOT/pages/about-api-catalog-cli.adoc

```diff
@@ -0,0 +1,689 @@
+
+=== Prerequisites
+
+* Download and install NodeJS and npm (node package manager) on your system. See the https://nodejs.org/en/download/[NodeJS download page^].
+
+=== Install the CLI
+
+To install the CLI:
+
+. Download the API Catalog CLI package from the https://npmjs.com[npm public registry^]. 
+. Run the following command from your terminal:
+----
+npm install -g api-catalog-cli@beta
+----
+
+=== View Command Help
+
+Use the help argument with the commands to list the commands or command options and their descriptions. For example:
+
+----
+api-catalog --help
+api-catalog autocomplete --help
+api-catalog create-descriptor --help
+api-catalog publish-asset --help
+----
+
+NOTE: You can optionally configure the `api-catalog autocomplete` command so that you can use the tab key with a partially-typed command to autocomplete it. See <<Configure Command Autocompletion>> for more details.
+
+== Configure Catalog Permissions in Anypoint Access Management
+
+You must enable the API Catalog Contributor permission for the Anypoint Platform organization user. 
+
+=== Prerequisites
+
+* The API Catalog entitlement must be set by MuleSoft for your Anypoint Platform organization. 
+
+NOTE: Permissions must be set by an Anypoint Platform user with administrative permission. 
+
+To enable the API Catalog Contributor permission:
+
+. In Anypoint Platform, navigate to *Access Management*.
+. Select *Users*.
+. Select the user for which you want to enable the permission. 
+. In the *Permissions* tab, select the *API Catalog* subtab.
+. Select *API Catalog Contributor*. 
+
+== Create a Descriptor File
+
+Before you run the commands to publish your APIs, you must create a descriptor file that has the information Exchange needs to catalog the APIs.  
+
+=== Create a Descriptor File Using the CLI
+
+You can run the `api-catalog create-descriptor` command to create and print the catalog descriptor YAML results to a designated file. The command will output cataloging information for all API specifications it finds in the full directory tree relative to the current working directory.
+
+To create a descriptor file using the CLI:
+
+. Navigate to the directory path that contains the APIs you want to catalog. 
+. Run the `api-catalog create-descriptor` command. For example:
++
+----
+> api-catalog create-descriptor -f mydescriptor.yaml
+----
+
+The result of the command depends on the presence and content of the designated descriptor file as follows:
+
+* No file or empty file: The command will discover the APIs and write the results to standard output and to a file named `catalog.yaml`.
+
+* Existing file with content: The command will fail and show an error message that says the descriptor file already exists.
+
+NOTE: You can optionally create a descriptor file manually instead of using the `api-catalog create-descriptor` command to generate the file. See <<Create a Descriptor File Manually>>.
+
+//Limitations:
```

Comment:
Yes, I wasn't able to get this information fleshed out, but left the comment here in case I get time to work with the Devs on this and/or get a request from the Beta customers to add limitations.